### PR TITLE
Update Vagrantfile to use PHP 5.6, to meet requirements of PHPUnit.

### PR DESCRIPTION
**Pull request by manish-makwana**
When trying out Kanboard with vagrant (V1.8.5) on macOS, I found the provisioning failed. This appears to be due to PHPUnit now requiring PHP 5.6+. I have made changes that get Kanboard working with a simple `vagrant up` now, and updated associated docs.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -1,7 +1,9 @@
 
 $script = <<SCRIPT
+sudo add-apt-repository ppa:ondrej/php
```
Comment (fguillot) [inline Vagrantfile]: I think Ubuntu Xenial have PHP 7 in the standard packages. The PPA should not be necessary, what about using php7?

**Comment by manish-makwana** [comment]
Sounds great, I'll test it out for PHP 7 and submit it again.

On 3/01/2018, at 8:09 AM, fguillot <notifications@github.com<mailto:notifications@github.com>> wrote:


@fguillot requested changes on this pull request.

________________________________

In Vagrantfile<https://github.com/kanboard/kanboard/pull/3647#discussion_r159293722>:

> @@ -1,7 +1,9 @@

 $script = <<SCRIPT
+sudo add-apt-repository ppa:ondrej/php


I think Ubuntu Xenial have PHP 7 in the standard packages. The PPA should not be necessary, what about using php7?

**Comment by fguillot** [comment]
Are you planning to update this pull-request?

**Comment by fguillot** [comment]
The Vagrantfile has been updated to PHP 7.

**Pull request closed without merging**